### ISSUE TITLE
Need to remove PORT from sonar_server.txt 

### ISSUE THREAD

**Issue by introllo** [JetBrains/TeamCity.SonarQubePlugin#39]
I have the sonar plugin working from TeamCity. However, the "View in sonar" link needs to be modified.

I've wrapped the SonarQube server with an Nginx reverse proxy (http --> https). The URL in the sonar_server.txt has the direct link to SonarQube web server (because the API cant handle the proxy).

The server URL (sonar.host.url) in the build step is http://my-domain:9000
Which generates a URL of http://my-domain:9000/dashboard/index/my_project for the results
I need the sonar_server.txt file to be http://my-domain/dashboard/index/my_project

As it stands, users are unable to login (forced login is ON) with the port spec. 

In short, I want the "View in sonar" link on the Build results page to go to the Nginx server *not* the Sonar web server.
